Packs/MandiantAdvantageThreatIntelligence/Integrations/MandiantAdvantageThreatIntelligence/MandiantAdvantageThreatIntelligence.py

```diff
@@ -1321,24 +1325,48 @@ def fetch_reputation(client: MandiantClient, args: dict = None):
 
     demisto.createIndicators(indicators)
 
+    output = []
     if indicators:
-        table = {
-            'Value': indicators[0]['value'],
-            'MScore': indicators[0]["rawJSON"].get("mscore", ''),
-            'Last Seen': indicators[0]["rawJSON"].get("last_seen", '')
-        }
-        indicator_type = indicators[0]["rawJSON"]["type"].lower()
-
-        markdown = tableToMarkdown(f'Mandiant Advantage Threat Intelligence information for {indicators[0]["value"]}\n'
-                                   f'[View on Mandiant Advantage](https://advantage.mandiant.com/indicator/'
-                                   f'{indicator_type}/{indicators[0]["value"]})',
-                                   table)
+        if input_type != "cve":
+            for indicator in indicators:
+                table = {
+                    'Value': indicator['value'],
+                    'MScore': indicator["rawJSON"].get("mscore", ''),
+                    'Last Seen': indicator["rawJSON"].get("last_seen", '')
+                }
+                indicator_type = indicator["rawJSON"]["type"].lower()
+
+                markdown = tableToMarkdown(f'Mandiant Advantage Threat Intelligence'
+                                           f'information for {indicator["value"]}\n'
+                                           f'[View on Mandiant Advantage](https://advantage.mandiant.com/indicator/'
+                                           f'{indicator_type}/{indicator["value"]})',
+                                           table)
+
+                output.append(CommandResults(
+                    readable_output=markdown,
+                    outputs=indicator,
+                    outputs_prefix=f"MANDIANTTI.{input_type.upper()}",
```

Comment:
you should create **one** `CommandResults` object as results 
and all the indicators should be in the outputs
you can check `Packs/FeedDHS/Integrations/DHS_Feed/DHS_Feed.py:get_indicators_results` as reference

in addition, as this are out of the initial context of this PR
it's very recommended to open a new PR for that

Reply:
Got it - I'll move that fix over to a new PR shortly.